src/screens/internal/CompareSnapshots/utils.ts

```diff
@@ -262,3 +264,43 @@ async function fetchTopCountiesByDiff(
     COUNTIES_LIMIT,
   ).map(rd => rd.region);
 }
+
+/**
+ * Maps a metric to the anomalies of its underlying data source.
+ * Outlier detection is applied to the raw data that is then used
+ * to calculate the metrics, so the relevant anomalies for each metric
+ * are the anomalies from the underlying data. E.g., for Weekly New Cases
+ * Per 100k we look at the anomalies in New Cases.
+ */
+export function getAnomaliesForMetric(
+  projection: Projections,
+  metric: Metric,
+): Anomalies | undefined {
+  assert(ALL_METRICS.includes(metric));
+  const annotations = projection.primary.annotations;
+  switch (metric) {
+    case Metric.WEEKLY_CASES_PER_100K ||
+      Metric.ADMISSIONS_PER_100K ||
+      Metric.CASE_GROWTH_RATE:
+      return annotations?.newCases?.anomalies;
```

Comment:
Per Slack conversation, I think ADMISSIONS_PER_100K shouldn't be in here, but CASE_DENSITY should.

Also I'm not sure if this syntax is correct.  I think `Metric.WEEKLY_CASES_PER_100K || Metric.ADMISSIONS_PER_100K || Metric.CASE_GROWTH_RATE` will just evaluate to the first truthy value, which will be `Metric.WEEKLY_CASES_PER_100K` and so it'll ignore the other items.

I think it needs to be something like:
```
case Metric.WEEKLY_CASES_PER_100K: 
case Metric.CASE_DENSITY:
case Metric.CASE_GROWTH_RATE:
  return annotations?.newCases?.anomalies;
```

Reply:
thanks for the catch and the explanation--updated in 142e41ea069eac537f5ed8968120603f4f7ac33f